### PR TITLE
GLT-1687: value corrections needed for FNS migration

### PR DESCRIPTION
**Pull request by djcooke**
After this is approved, will run this on prod, followed by the changed lines:
```
UPDATE KitDescriptor SET name = '10X Single Cell RNA' WHERE name = '10X scRNA';
DELETE FROM Indices WHERE indexFamilyId = (SELECT indexFamilyId FROM IndexFamily WHERE name = '10X Family');
```

* 10X Family barcodes actually have 4 sequences. We are going to leave it up to analysis to sort this out based on the index name
* Changed the kit name to match the existing kit name from GSLE

### CODE REVIEW

File: sqlstore/src/main/resources/db/migration_afterMigrate/60_indices.sql [hunk 1]
```diff
@@ -2,6 +2,101 @@
 UPDATE IndexFamily SET platformType = 'ILLUMINA' WHERE name = '454' AND platformType = 'LS454';
 
 CALL addIndexFamily('10X Family', 'ILLUMINA', FALSE);
-CALL addIndex('10X Family', 'SI-GA-H8', 'TTGTTGAT', 1);
-CALL addIndex('10X Family', 'SI-GA-H9', 'ACACTGTT', 1);
+CALL addIndexFamily('10X Family', 'ILLUMINA', FALSE);
```
Comment (apmasell) [inline sqlstore/src/main/resources/db/migration_afterMigrate/60_indices.sql]: This line seems duplicated.

File: sqlstore/src/main/resources/db/migration_afterMigrate/60_indices.sql [hunk 2]
```diff
@@ -2,6 +2,101 @@
...
+CALL addIndex('10X Family', 'SI-GA-A1', 'SI-GA-A1', 1);
```
Comment (apmasell) [inline sqlstore/src/main/resources/db/migration_afterMigrate/60_indices.sql]: Please put in a comment here about these being non-sequence barcodes.

**Review by apmasell — APPROVED** [review]

File: sqlstore/src/main/resources/db/migration_afterMigrate/60_indices.sql (hunk 1 above)
Comment (djcooke) [reply sqlstore/src/main/resources/db/migration_afterMigrate/60_indices.sql]: removed

File: sqlstore/src/main/resources/db/migration_afterMigrate/60_indices.sql (hunk 2 above)
Comment (djcooke) [reply sqlstore/src/main/resources/db/migration_afterMigrate/60_indices.sql]: added

**Pull request merged**